packages/core/src/common/performance/stopwatch.ts

```diff
@@ -50,10 +51,20 @@ export abstract class Stopwatch {
     @inject(ILogger)
     protected readonly logger: ILogger;
 
+    protected cachedResults: MeasurementResult[] = [];
+
+    protected onMeasurementResultEmitter = new Emitter<MeasurementResult>();
```

Comment:
According to our [guidelines](https://github.com/eclipse-theia/theia/blob/master/doc/coding-guidelines.md#event_names), the name of this event should be `onDidAddMeasurementResult`.